python/fusion_engine_client/messages/control.py

```diff
@@ -176,41 +176,94 @@ class ResetRequest(MessagePayload):
     ##
     # Perform a device hot start.
     #
-    # A hot start is typically used to restart the navigation engine in a
-    # deterministic state, particularly for logging purposes.
+    # This will reset the navigation engine into a known state, using previously
+    # stored position and time information. The device will begin navigating
+    # immediately if possible.
     #
     # To be reset:
     # - The navigation engine (@ref RESTART_NAVIGATION_ENGINE)
-    # - All runtime data (GNSS corrections (@ref RESET_GNSS_CORRECTIONS), etc.)
     #
-    # Not reset:
+    # Not reset/performed:
+    # - All runtime data (GNSS corrections (@ref RESET_GNSS_CORRECTIONS), etc.)
     # - Position, velocity, orientation (@ref RESET_POSITION_DATA)
+    # - GNSS ephemeris data (@ref RESET_EPHEMERIS)
     # - Calibration data (@ref RESET_CALIBRATION_DATA)
     # - User configuration settings (@ref RESET_CONFIG)
-    # - GNSS measurement engine (@ref RESTART_GNSS_MEASUREMENT_ENGINE)
+    # - Reboot GNSS measurement engine (@ref RESTART_GNSS_MEASUREMENT_ENGINE)
     # - Reboot navigation processor (@ref REBOOT_NAVIGATION_PROCESSOR)
-    HOT_START = 0x000000FF
+    HOT_START = 0x00000001
 
     ##
     # Perform a device warm start.
     #
-    # A warm start is typically used to reset the device's estimate of position
-    # and kinematic state in case of error.
+    # During a warm start, the device retains its knowledge of approximate
+    # position and time, plus almanac data if available, but resets all ephemeris
+    # data. As a result, the device will need to download ephemeris data before
+    # continuing to navigate with GNSS.
+    #
+    # To be reset:
+    # - The navigation engine (i.e., perform a hot start, @ref
+    #   RESTART_NAVIGATION_ENGINE)
+    # - GNSS ephemeris data (@ref RESET_EPHEMERIS)
+    #
+    # Not reset/performed:
+    # - All runtime data (GNSS corrections (@ref RESET_GNSS_CORRECTIONS), etc.)
+    # - Position, velocity, orientation (@ref RESET_POSITION_DATA)
+    # - Fast IMU corrections (@ref RESET_FAST_IMU_CORRECTIONS)
+    # - Training parameters (slowly estimated IMU corrections, temperature
+    #   compensation, etc.; @ref RESET_NAVIGATION_ENGINE_DATA)
+    # - Calibration data (@ref RESET_CALIBRATION_DATA)
+    # - User configuration settings (@ref RESET_CONFIG)
+    # - Reboot GNSS measurement engine (@ref RESTART_GNSS_MEASUREMENT_ENGINE)
+    # - Reboot navigation processor (@ref REBOOT_NAVIGATION_PROCESSOR)
+    WARM_START = 0x00000201
+
```

Comment:
Descriptions for the below methods (`PVT_RESET`, `DIAGNOSTIC_LOG_RESET`, and `COLD_START`) have populated the full 10 items that I think coordinated.